cmd/libs/go2idl/client-gen/generators/client_generator.go

```diff
@@ -217,6 +217,57 @@ func Packages(context *generator.Context, arguments *args.GeneratorArgs) generat
 	}
 	includedTypesOverrides := customArgs.IncludedTypesOverrides
 
+	// If there is a comment of the form "// +groupName=somegroup" or
```

Comment:
I can do a function, yes.

I don't believe I can move to main.go because I need access to `context.Universe` so I can get the package and its `DocComments`, which aren't available in main.go.